src/ophys_etl/transforms/mesoscope_2p/tiff.py

```diff
@@ -262,11 +265,21 @@ def volume_scans(self):
 
     @property
     def plane_stride(self):
-        return len(self.plane_scans)
+        stride = 0
+        # only increment stride for unique zs, because same z end up in the
```

Comment:
please replace block comment with docstring, indicate return type and add return type hint